### DISCUSSION TITLE
Page deletion through git repo

### DISCUSSION THREAD

**Discussion by paolofacchinetti** [requarks/wiki#6327] [Help / Questions]
In my wikijs installation I have git storage enabled.

I wanted to delete a bunch of pages and came across [issue #195](https://github.com/Requarks/wiki/issues/195)

I tried deleting the pages from the git repository, expecting them to disappear from the wiki after a sync. Unfortunately, I can still the pages in the webapp and in the Postgres DB, while the git repository and the wikijs local storage (inside the `/wiki/data/repo` dir) do not have the pages i deleted.

If i try to delete the pages manually from the UI, I get an error that says:

`fatal: pathspec '<FILE_PATH>' did not match any files`

Should i just go through the database and delete pages, assets, etc? There has to be a more efficient way

**Comment by ghost** [comment]
I'm having a similar issue. Were you ever able to resolve this?
